specification/sbi_atee.adoc

```diff
@@ -13,6 +43,858 @@ running a VMM).
 The following operations must be implemented by the TSM ( _Formal input 
 parameters and return codes are TBD_ ).
 
+== TEE Extension (EID #0x41544545)
+
+=== Function: Create TVM (FID #0)
+[source, C]
+-----
+struct sbiret sbi_tee_create_tvm(unsigned long tvm_create_params,
+                                 unsigned long tvm_create_params_len);
+-----
+Creates a confidential TVM using the specified parameters. The tvm_create_parameters
+is a pointer to the `tvm_create_params` described below, and tvm_create_params_len is
+the size of the structure in bytes. 
+
+Callers of this API should first invoke `sbi_tee_tsm_get_info()` to obtain information
```

Comment:
I think sbi_tee_tsm_get_info should be FID #0 so that the reader is already aware of the function. There are multiple reference to tsm_info in this function.

Reply:
I think chronological sequence more sense in the spec as it helps coherent reading. All other SBI extensions assign FIDs in that order as well.

Reply:
I have taken a stab at it, and have added more exposition...PTAL.